storage/rocksdb/ha_rocksdb.cc

```diff
@@ -10288,8 +10288,15 @@ int ha_rocksdb::create_table(const std::string &table_name,
   DBUG_ENTER_FUNC();
 
   int err;
+  // DD tables should create in __system CF.
+  // During DDSE change, sql layer will create DD tables as <upgrade>.<table>
+  // first, then rename it to mysql.<table>.
+  // rename is meta-data only change, thus treat these <upgrade>.<table>
+  // as DD table.
+  auto db_name =
+      ha_thd()->is_dd_system_thread() ? "mysql" : table_arg.s->db.str;
```

Comment:
Is the `is_dd_system_thread` check granular enough? I am a bit worried about this thread creating e.g. system not DD tables at a later point.

Reply:
if it is system table, then the later code --- dd::get_dictionary()->is_dd_table_name() will return false, since it isn't DD table.

Reply:
OK